ingestion/src/metadata/utils/workflow_output_handler.py

```diff
@@ -242,3 +242,33 @@ def print_test_suite_status(workflow) -> None:
         click.secho("Workflow finished with failures", fg="bright_red", bold=True)
     else:
         click.secho("Workflow finished successfully", fg="green", bold=True)
+
+
+def print_data_insight_status(workflow) -> None:
+    """Runs click echo to print the test suite workflow results
+
+    Args:
+        workflow (DataInsightWorkflow): workflow object
+    """
+    click.echo()
+    click.secho("Processor Status:", bold=True)
```

Comment:
we're moving away from click in https://github.com/open-metadata/OpenMetadata/pull/8182, let's not add it to new changes 🙏